next.json

```diff
@@ -535,6 +634,12 @@
                 "args": [
                     {"name": "input", "type": "input state"}
                 ]
+            },
+            {
+                "name": "set depth stencil state",
+                "args": [
+                    {"name": "input", "type": "depth stencil state"}
```

Comment:
nit: `"name": "state"` or something. ("input" was probably also a bad name for the argument to SetInputState, my bad)